### PR TITLE
Created get-all-products

### PR DESCRIPTION
**Pull request by nuinn**
Products router created
Video-games router deleted
getAll function created which returns all products by most recent

**PACKAGE LOCK HAS SOME CONFLICTS, NOT SURE IF WE CAN AVOID PUSHING THIS TO THE DEVELOP BRANCH**

### CODE REVIEW

**Review by Ahuanmarca — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 node_modules
 .env
 coverage
+package-lock.json
```
Comment (JonaTheBridge) [inline .gitignore]: esto no debería ir en .gitignore porque debemos tener todos las mismas versiones

**Review by JonaTheBridge — CHANGES_REQUESTED** [review]

**Review by andreamir — APPROVED** [review]
👍🏻

**Review by jaanmeld — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (andreamir) [inline .gitignore]: ﹢1️⃣

**Review by andreamir — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (nuinn) [reply .gitignore]: vale, cambiado

**Pull request merged**